.gitignore

```diff
@@ -107,3 +107,5 @@ media/
 
 # Generated scripts
 bin/archived_cleanup.sh
+
+frontend/.vscode/
```

Comment:
@giantas, move this to [this line](https://github.com/open-craft/opencraft/blob/master/.gitignore#L70) and put a comment specifying its vscode. Follow the convention in the file. Also, rather than putting `frontend/.vscode/`, `.vscode/` will be better. What do you think?

Reply:
I agree. It seems `.vscode/` also ignores `frontend/.vscode/` which was the intention. Thank you for this.